packages/cactus-test-plugin-htlc-eth-besu/src/test/typescript/integration/plugin-htlc-eth-besu/new-contract-endpoint-invalid.test.ts

```diff
@@ -177,11 +177,9 @@ describe(testCase, () => {
         keychainId: "",
         gas: DataTest.estimated_gas,
       };
-      const resp = await api.newContractV1(bodyObj);
-      expect(resp).toBeTruthy();
-      expect(resp.status).toEqual(200);
+      await api.newContractV1(bodyObj);
```

Comment:
@Leeyoungone This would make the test pass in the event of the API call (incorrectly) succeeding with the invalid input.

Please do something like this instead to assert the failure:

```typescript
    const bodyObj: NewContractObj = {
      contractAddress: hashTimeLockAddress,
      inputAmount: 10,
      outputAmount: 0x04,
      expiration: DataTest.expiration,
      hashLock: DataTest.hashLock,
      receiver: DataTest.receiver,
      outputNetwork: "BTC",
      outputAddress: "1AcVYm7M3kkJQH28FXAvyBFQzFRL6xPKu8",
      connectorId: connectorId,
      web3SigningCredential,
      keychainId: "",
      gas: DataTest.estimated_gas,
    };
    const apiCall = api.newContractV1(bodyObj);
    await expect(apiCall).rejects.toHaveProperty(["response", "status"], 500);
```

Reply:
Got it ! Thank you :)